### DISCUSSION TITLE
Unit tests for Oracle

### DISCUSSION THREAD

**Discussion by FaritSlv** [doctrine/doctrine2#9861] [General]
Hi, @derrabus! 

Tell me, please, do you specifically bypass unit tests for Oracle DB or is there no one to do this?
I ran a couple of times and a lot of errors came out

**Comment by greg0ire** [comment]
Here is my latest attempt regarding this: https://github.com/doctrine/orm/pull/8958 (so yes we do bypass them, or rather, there is no build for Oracle).

**Reply by FaritSlv** [reply]
DBAL has a normal assembly for ORACLE, it does not work out in a minute, of course, but within 5 minutes it seems to fit.

I also used your builds

**Reply by greg0ire** [reply]
What do you mean by "a normal assembly"? Feel free to pick up my PR and fix whatever is left to fix.

**Reply by FaritSlv** [reply]
I mean, the assembly does not take 30 minutes. OK. I'll try to edit your branch

**Reply by greg0ire** [reply]
Ok, what do you mean by "assembly" :thinking: ?

**Reply by FaritSlv** [reply]
Maybe I'm wrong. But it seems to me that because of the assembly there was a long execution of tests. 

Now I'll try to figure it out

**Reply by greg0ire** [reply]
I don't know if you are right or wrong, I just don't know what you mean by assembly… do you mean assembly as in instructions for the CPU?

**Reply by FaritSlv** [reply]
No.
The first launch took about 10 minutes.
Hangs due to some timeout in tests [#16](https://github.com/FaritSlv/orm/actions/runs/2558665263)
